src/ui/localization/locale/en.json

```
@@ -0,0 +1,56 @@
+{
+  "periods.week": "{{amount}} week",
+  "periods.month": "{{amount}} month",
+  "periods.year": "{{amount}} year",
+  "periods.day": "{{amount}} day",
+  "periods.weekShort": "{{amount}}wk",
```

Hmm maybe it's ok as is but would it be ok to add a delimiter like
```suggestion
  "periods.weekShort": "{{amount}}/wk",
```
Same for the other short periods. Not sure if we want to go the really short way though.

---

I think those are used in a variable string that already has delimiters so they would look like
```
3$/{{amount}}wk
```

The {{amount}} is the number of weeks, not the money per week, maybe amount is not the best name for that variable.